_maps/map_files/RandomRuins/LavaRuins/lavaland_surface_ufo_crash.dmm

```diff
@@ -0,0 +1,431 @@
+//MAP CONVERTED BY dmm2tgm.py THIS HEADER COMMENT PREVENTS RECONVERSION, DO NOT REMOVE
+"a" = (
+/turf/simulated/mineral/volcanic/lava_land_surface,
+/area/lavaland/surface/outdoors)
+"b" = (
+/turf/simulated/floor/plating/asteroid/basalt/lava_land_surface,
+/area/lavaland/surface/outdoors)
+"c" = (
+/obj/machinery/abductor/experiment,
+/turf/unsimulated/floor/abductor,
+/area/ruin/unpowered)
+"d" = (
+/turf/simulated/wall/mineral/abductor,
+/area/ruin/unpowered)
+"e" = (
+/turf/unsimulated/floor/abductor,
+/area/ruin/unpowered)
+"g" = (
+/obj/machinery/abductor/pad,
+/turf/unsimulated/floor/abductor,
+/area/ruin/unpowered)
+"h" = (
+/obj/item/hemostat/alien,
+/turf/unsimulated/floor/abductor,
+/area/ruin/unpowered)
+"i" = (
+/obj/effect/mob_spawn/human/abductor,
+/turf/unsimulated/floor/abductor,
+/area/ruin/unpowered)
+"j" = (
+/obj/structure/closet/abductor,
+/turf/unsimulated/floor/abductor,
+/area/ruin/unpowered)
+"k" = (
+/obj/effect/baseturf_helper/lava_land/surface,
+/turf/unsimulated/floor/abductor,
+/area/ruin/unpowered)
+"l" = (
+/obj/machinery/optable/abductor,
+/obj/item/cautery/alien,
+/turf/unsimulated/floor/abductor,
+/area/ruin/unpowered)
+"m" = (
+/obj/structure/table/abductor,
+/obj/item/storage/box/alienhandcuffs,
+/turf/unsimulated/floor/abductor,
+/area/ruin/unpowered)
+"n" = (
+/obj/item/scalpel/alien,
+/obj/item/surgical_drapes,
+/turf/unsimulated/floor/abductor,
+/area/ruin/unpowered)
+"o" = (
+/obj/item/retractor/alien,
+/turf/unsimulated/floor/abductor,
+/area/ruin/unpowered)
+"p" = (
+/obj/machinery/abductor/gland_dispenser,
```

Comment:
Seems like the crashed mothership still has the organ dispenser to me.

Reply:
Ah. From the looks of it it doesn't use power.